src/state/opamSwitchState.mli

```diff
@@ -54,6 +54,13 @@ val compute_available_packages:
   pinned:package_set -> opams:OpamFile.OPAM.t package_map ->
   package_set
 
+(** Raw function to compute the conflicts for all packages, given
+    the set of available packages and the corresponding opam files.
+    the `u_conflicts` field when building a universe manually. *)
```

Comment:
Is there a missing line in this ocamldoc comment?  `the u_conflicts field <...>` seems to be incomplete.

Reply:
Not sure where that went, fixed thanks.